src/components/book/BookContentCard.js

```diff
@@ -54,51 +49,47 @@ const styles = {
 
 class BookContentCard extends Component {
   render() {
-    const { bookImage, author, title, desc, userImage } = this.props;
+    const { bookImage, author, title, desc, userImage, updatedAt } = this.props;
     const { classes } = this.props;
     return (
       <Fragment>
         <Card className={classes.root}>
           <CardActionArea>
-            <CardMedia
-              className={classes.media}
-              image={bookImage}
-              title={title}
-            />
-            <CardContent className={classes.content}>
-              {/* <Typography gutterBottom variant="subtitle1" component="span" color="primary">
-                {author}
-              </Typography> */}
-              <Typography
-                className={classes.title}
-                gutterBottom
-                variant="h6"
-                component="h6"
-                align="center"
-              >
-                {formatStringThumbnail(title)}
-              </Typography>
-              <Typography
-                className={classes.desc}
-                variant="body2"
-                color="textSecondary"
-                component="p"
-              >
-                {formatString(desc.replace(/(<([^>]+)>)/g, ""))}
-              </Typography>
-            </CardContent>
-          </CardActionArea>
-          <CardActions className={classes.cardActions}>
-            <Button size="small" className={classes.button}>
-              <ShareIcon />
-            </Button>
             <BookContentCardDetail
               bookImage={bookImage}
               author={author}
               title={title}
               desc={desc.replace(/(<([^>]+)>)/g, "")}
               userImage={userImage}
+              updatedAt={updatedAt}
             />
+          </CardActionArea>
+          <CardContent className={classes.content}>
+            {/* <Typography gutterBottom variant="subtitle1" component="span" color="primary">
+              {author}
+            </Typography> */}
+            <Typography
+              className={classes.title}
+              gutterBottom
+              variant="h6"
+              component="h6"
+              align="center"
+            >
+              {formatStringThumbnail(title)}
+            </Typography>
+            <Typography
+              className={classes.desc}
+              variant="body2"
+              color="textSecondary"
+              component="p"
+            >
+              {formatString(desc.replace(/(<([^>]+)>)/g, ""))}
```

Comment:
maybe we should make a new method in the util.js file for getting rid of html code? I feel like this could be handy for later!

Reply:
That is right. I will do that.

Reply:
I made a new method to get ride of html tag.